src/integration/common/index.js

```diff
@@ -11,13 +11,15 @@ import runPerformanceTests from './performance';
 import runRadioScheduleTests from './radioSchedule';
 import runCrossPlatformSEOTests from './SEO';
 import runSeoCanonicalTests from './SEO.canonical';
+import runStoryPromoTests from './storyPromo';
 
 const runCommonCrossPlatformTests = service => {
   runA11yTests();
   runHeaderTests(service);
   runFooterTests();
   runPerformanceTests();
   runCrossPlatformSEOTests();
+  runStoryPromoTests();
```

Comment:
This should probably not belong here, as not all page types will have story promos. Instead, let's move it to the export section, and import it where we need to check for Story Promos